modules/progress/views/progress.html

```diff
@@ -1,6 +1,4 @@
-<div class="progress-container progress-container--{{ lxProgress.lxType }} progress-container--{{ lxProgress.lxColor }}"
-     ng-class="{ 'progress-container--determinate': lxProgress.lxValue,
```

Comment:
Try checking if `undefined` or not instead of auto-typing this to a boolean. There will be a problem in the case `lxValue` is `0`.
This may solve your problem without using a ternary in an interpolation

Reply:
Definitely checking for `undefined` might solve the problem, I'll try it locally and comment accordingly.

Reply:
`ng-class` still not triggered by the value of `lxValue`, and option would be maybe check for `lxValue` in controller, thereby create a new property `lxProgress.lxClass`, like so:
 
```
lxProgress.lxClass = angular.isDefined(lxProgress.lxValue) ? 'progress-container--determinate' : 'progress-container--indeterminate';
```

```
<div class="progress-container progress-container--{{ lxProgress.lxType }} progress-container--{{ lxProgress.lxColor }} {{ lxProgress.lxClass }}">
```